internal/checker/checker.go

```diff
@@ -25833,6 +25833,73 @@ func (c *Checker) markPropertyAsReferenced(prop *ast.Symbol, nodeForCheckWriteOn
 	c.symbolReferenceLinks.Get(target).referenceKinds |= ast.SymbolFlagsAll
 }
 
+func (c *Checker) GetExpandedParameters(signature *Signature /* !!! skipUnionExpanding */) []*ast.Symbol {
```

Comment:
Should this be returning `[][]*ast.Symbol` to match the old code?

Reply:
Not yet - unless u want me to implement `skipUnionExpanding` right away. I skipped it since it’s only required by services and there are no tests for them yet. I could ofc just do this preemptively but it felt weird to do so

Reply:
I guess it's fine; anyone going to add that param will realize they need to modify it.